test/error.js

```diff
@@ -33,4 +34,14 @@ describe('error handling', () => {
     assert.throws(humanizing({ language: null }), Error)
     assert.doesNotThrow(humanizing({ language: 'es' }), Error)
   })
+
+  it('does not throw an error when passed a bad language in the function and a valid fallback specified', () => {
```

Comment:
I'd rather have this test the actual functionality, rather than that it doesn't throw. For example:

```js
it('accepts fallback languages', () => {
  // ...

  assert.strictEqual(humanizingWith({ language: 'es', fallback: 'en' }), '10 segundos')
  assert.strictEqual(humanizingWith({ language: 'BAD', fallback: 'en' }), '10 seconds')
})
```

The test would need to go in another file, of course.

Reply:
I don't think this comment has been addressed.

Reply:
Invalid configuration covered in error tests. 

Functionality tests are in tests/humanizer under `accepts fallback languages`